crates/summit/src/task.rs

```diff
@@ -212,12 +213,18 @@ pub async fn create_missing(
 pub async fn set_status(tx: &mut Transaction, task_id: task::Id, status: Status) -> Result<()> {
     let ended = if !status.is_open() { ", ended = unixepoch()" } else { "" };
 
+    let started = if status == Status::Building {
+        ", started = unixepoch()"
+    } else {
+        ""
+    };
+
```

Comment:
It's possible for a task to get requeued (set to New again) and so we should set `started` back to null in those instances